crates/air_r_formatter/tests/specs/r/air.toml

```diff
@@ -0,0 +1,2 @@
+[format]
+exclude=["**/*.R"]
```

Comment:
I think you should remove this file

Currently Air does not recurse into folders to find `air.toml`, so having this here has no effect unless you literally open the VS Code project at `tests/specs/r/`, which we don't do

So I think it is just confusing because saving in the R files here still rearranges them

Reply:
hmm I must have gotten confused and disabled formatting in my setup for these files in two different ways